ui/app/AppLayouts/Chat/panels/communities/CommunityMembersSettingsPanel.qml

```diff
@@ -95,7 +95,8 @@ SettingsPageLayout {
                 id: memberItem
 
                 readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
-                readonly property bool isOnline: model.onlineStatus === Constants.userStatus.online
+                readonly property bool isOnline: memberItem.itsMe ? true
```

Comment:
This condition seems to be invalid.  Assumption that "me" is always online is probably wrong :thinking:

Reply:
@osmaczko actually I don't know how it should be here. I can process every each status or process `online/offline`. There was be used just `online`. What do you think?

Reply:
Not sure what you mean.. `membersModel` should be already populated with proper status, no extra logic is needed here.